examples/sycl/decision_forest_classification_batch.py

```diff
@@ -0,0 +1,143 @@
+
+    # now predict using the model from the training above
+    if daal_version < (2020,1):
+        predict_algo = d4p.decision_forest_classification_prediction(nClasses=5)
+    else:
+        predict_algo = d4p.decision_forest_classification_prediction(nClasses=5,
+            resultsToEvaluate="computeClassLabels|computeClassProbabilities", votingMethod="unweighted")
+
+    predict_result = predict_algo.compute(predict_data, train_result.model)
+
+    return train_result, predict_result
+
+# At this moment with sycl we are working only with numpy arrays
+def to_numpy(data):
+    try:
+        from pandas import DataFrame
+        if isinstance(data, DataFrame):
+            return np.ascontiguousarray(data.values)
+    except Exception as e:
+        pass
+    try:
+        from scipy.sparse import csr_matrix
+        if isinstance(data, csr_matrix):
+            return data.toarray()
+    except Exception as e:
+        pass
+    return data
+
+def main(readcsv=read_csv, method='defaultDense'):
+    nFeatures = 3
+    # input data file
+    train_file = os.path.join('..', 'data', 'batch', 'df_classification_train.csv')
+    predict_file = os.path.join('..', 'data', 'batch', 'df_classification_test.csv')
+
+    # Read train data. Let's use 3 features per observation
+    train_data   = readcsv(train_file, range(nFeatures), t=np.float32)
+    train_labels = readcsv(train_file, range(nFeatures,nFeatures + 1), t=np.float32)
+    # Read test data (with same #features)
+    predict_data = readcsv(predict_file, range(nFeatures), t=np.float32)
+    predict_labels = readcsv(predict_file, range(nFeatures, nFeatures + 1), t=np.float32)
+
+    # Using of the classic way (computations on CPU)
+    train_result, predict_result = compute(train_data, train_labels, predict_data, "defaultDense")
+    assert predict_result.prediction.shape == (predict_labels.shape[0], 1)
+    assert (np.mean(np.not_equal(predict_result.prediction - predict_labels, 0)) < 0.03).any()
+
+    train_data   = to_numpy(train_data) 
+    train_labels = to_numpy(train_labels)
+    predict_data = to_numpy(predict_data)
+
+    try:
+        from dpctx import device_context, device_type
+        gpu_context = lambda: device_context(device_type.gpu, 0)
+    except Exception as e:
+        from daal4py.oneapi import sycl_context
+        gpu_context = lambda: sycl_context('gpu')
+
+    # It is possible to specify to make the computations on GPU
+    if gpu_available:
+        with gpu_context():
+            sycl_train_data = sycl_buffer(train_data)
+            sycl_train_labels = sycl_buffer(train_labels)
+            sycl_predict_data = sycl_buffer(predict_data)
+            train_result, predict_result = compute(sycl_train_data, sycl_train_labels, sycl_predict_data, 'hist')
+            assert predict_result.prediction.shape == (predict_labels.shape[0], 1)
+            assert (np.mean(np.not_equal(predict_result.prediction - predict_labels, 0)) < 0.03).any()
```

Comment:
```suggestion
            assert (np.mean(predict_result.prediction == predict_labels)) < 0.03).any()
```

Reply:
thank you, replaced with np.mean(predict_result.prediction != predict_labels)